doc/source/visualizing/geographic_projections_and_transforms.rst

```diff
@@ -0,0 +1,111 @@
+.. _geographic_projections_and_transforms:
+
+Geographic Projections and Transforms
+=====================================
+
+Geographic data that is on a sphere can be visualized by projecting that data
+onto a representation of that sphere flattened into 2d space. There exist a
+number of projection types, which can be found in the `the cartopy
+documentation <https://scitools.org.uk/cartopy/docs/v0.15/crs/projections.html>`_.
+With support from `cartopy <https://scitools.org.uk/cartopy/docs/latest/>`_, 
+``yt`` now supports these projection 
+types for geographically loaded data.
+Underlying data is assumed to have a transform of `PlateCarree
+<https://scitools.org.uk/cartopy/docs/v0.15/crs/projections.html#platecarree>`_,
+which is data on a flattened, rectangular, latitude/longitude grid. The data
+transform is what system your data is defined with and the data projection is
+what the resulting plot will display. For more information on this difference,
+refer to `the cartopy documentation on these differences
+<https://scitools.org.uk/cartopy/docs/latest/tutorials/understanding_transform.html>`_. 
+If your data is not of this form, feel free to open an issue or file a pull
+request on the ``yt`` github page for this feature.
+
+It should be noted that
+these projections are not the same as yt's ProjectionPlot. For more information
+on yt's projection plots, see :ref:`projection-types`.
+
+Installation
+^^^^^^^^^^^^
+
+In order to access the geographic projection functionality, you will need to have an
+installation of ``cartopy`` available on your machine.
+
+.. code-block:: bash
+
+    conda install cartopy
+
+Using Basic Projections
+^^^^^^^^^^^^^^^^^^^^^^^
+
+All of the transforms available in ``Cartopy`` v0.15 are accessible with this
+functionality.
+
+The next few examples will use a GEOS dataset accessible from the ``yt`` data
+downloads page. For details about loading this data, please 
+see :ref:`cookbook-geographic_projections`.
+
+If a geographic dataset is loaded without any defined projection the default
+option of `PlateCarree` will be displayed.
+
+.. code-block:: python
+
+    ds = yt.load_uniform_grid(data, sizes, 1.0, geometry=("geographic", dims),
+    bbox=bbox)
+    p = yt.SlicePlot(ds, "altitude", 'AIRDENS')
+
+If an option other than `PlateCarree` is desired, the plot projection type can
+be set manually either in the arguments for `SlicePlot` or 
+with the `set_mpl_projection` function. The next two code blocks show how to 
+sets the projection to a `Robinson` projection from the default `PlateCarree`
+using either method.
+
+.. code-block:: python
+
+    ds = yt.load_uniform_grid(data, sizes, 1.0, geometry=("geographic", dims),
+    bbox=bbox)
+    p = yt.SlicePlot(ds, "altitude", 'AIRDENS', geo_projection="Robinson")
```

Comment:
I must confess I'm not *hugely* keen on this specifically being `geo_projection`, since we may want to extend it to things that aren't necessarily "geographic," but at the same time, keeping it distinct from the (yt-centric) notion of a `ProjectionPlot` is desirable.  I wonder if there's a way to balance this with something like `plot_projection` or something?

Reply:
I agree. I struggled choosing a name for this variable so I have no strong feelings about `geo_projection`. I do think `plot_projection` works and I'd be fine with changing it to that. Another thought that I had was that if we build in support for transforms other than `PlateCarree` we are going to need to add in an argument for the transform too. We might want these variables have complementary naming conventions so their relationship is obvious. I'm not sure `plot_transform` would be totally accurate since it describes more about the data than the plot, so I might lean towards `data_transform` for that. What if we used `data_projection`?

Reply:
So about passing in transforms and projections to the initializers - do we really need to do that? I think we already have enough arguments to the initializers and if we added arguments every time we added new functionality that list would just get longer and longer. Is it really so un-ergonomic to have a setter? Keep in mind that when we detect we're loading a geographic dataset we can just choose a default projection and use that, allowing users to customize it by calling the setter or by adding a new configuration option.

Reply:
I'd be fine with not passing it in as long as we have an implementation for a default projection.  But we don't right now, so I'm less sure that anyone would even see this if we don't put it in the constructor.

@munkm how hard would it be to make the default projection grabbed from the coordinate handler?  I think we could have it be a property like `axis_name` or whatever, and just return a string based on the axis being viewed.

Reply:
re: default projections -- Matt and I talked offline about this and I'm setting up something in the coordinate handler (specifically in the base handler and the geographic coordinate handler) to do this. The defaults set in the coordinate handler will be called to set `self._transform` and `self._projection` in `PlotWindow`. 

@ngoldbaum @MatthewTurk I had a thought about passing the arguments to the initializers. I'm on board with removing this for the projection argument. However, the transform argument makes more sense to send through the initializer. There's no reason that somebody would want to choose or change the data transform to be anything other than the transform that inherently describes the data. I can set a default transform in the coordinate handler, but if a user wants to override that, there's no way to do that (yet). What if I modify the existing argument in the initializer to set the transform, and not the projection?

Reply:
Madicken can we chat in person about this issue? I'm a little confused and it will be easier to work through that in person than via github issues.